components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -149,12 +137,6 @@ class UserLimitAnnouncementBar extends React.PureComponent<Props> {
             return null;
         }
 
-        // If AT user limit, and banner hidden, don't render anything
-        if (userLimit === analytics!.TOTAL_USERS &&
-            preferences.some((pref) => pref.name === CloudBanners.HIDE && pref.value === 'true')) {
-            return null;
-        }
-
         const dismissable = this.isDismissable();
```

Comment:
Nit: can remove `isDismissable` method, and have the `message=` down below always be `t('upgrade.cloud_banner_reached')`